apps/docs/guide/running/running-application.md

```diff
@@ -16,6 +16,21 @@ The private chain by default has a block time of 5 seconds, controlled by `--blo
 
 By default the node closes an epoch once a day, but this can be controlled by the `--epoch-duration <seconds>` command option. It's an important settings when it comes down to voucher execution.
 
+## Rollups Node configuration
+
+You can create a `.sunodo.env` file in the root of your project to configure the node.
+
+All Rollups Node services can be configured using environment variables.
+
+For ex., in case you want to change the default deadline for adanving the state for the rollups-advance-runner service.
+
+```shell
+cat .sunodo.env
+SM_DEADLINE_ADVANCE_STATE=360000
+```
+
+Check the configuration options for each service in the [Rollups Node documentation](https://github.com/cartesi/rollups/blob/v1.0.1/offchain/README.md).
```

Comment:
ps: except that it's not documented there

Reply:
yeah, what do you suggest?

I thought on instructing the user to nu `docker run ... cartesi/rollups-advance-runner --help` but I felt it's out of scope for this documentation, and ugly.